### ISSUE TITLE
supplement: question about "formal equation" for Bayes act

### ISSUE THREAD

**Issue by elray1** [aaronger/utility-eval-papers#69]
Here's the text in question:
![image](https://github.com/aaronger/utility-eval-papers/assets/5132208/c7458057-e8b1-44b6-8b36-5f6ee19f27c7)

 - My question is whether we need to include Eq (30)?
 - Motivations:
     - it took me a while to process what was going on here because I didn't catch all the notation, and had to think through "the point is that the left hand side gives you the same value, $\tau$, repeated $N$ times, and that matches the $\tau$ that's returned by $F_T(K)$".
     - Is this notation needed to set up (31)? It doesn't seem to be used there, and we could already just write down (31) based on (28) and (27), right?
![image](https://github.com/aaronger/utility-eval-papers/assets/5132208/5467bc68-c25f-41b4-821e-b1d34d94f4c5)

**Comment by aaronger** [comment]
Yes, we already know (31) so (30) is not logically carrying much weight.  But I find the 1_A adjoint idea to be helpful as offering another perspective and leading perhaps to a generalization to other "totalling" maps.  But I can remove it if it seems too distracting.

**Comment by elray1** [comment]
I vote for removing it here -- the cognitive burden on the reader making their way through this supplement is already high, so I think we should try to avoid extra things :)

**Comment by aaronger** [comment]
OK will try to do soon, but the $\undertilde{\bm{F}}$ is starting to feel like a better solution than anything I was using before.  Maybe I set myself up by saying "interesting"...
